src/sid/simulate.py

```diff
@@ -268,12 +269,6 @@ def get_simulate_func(
     )
 
     if _are_states_prepared(initial_states):
-        if initial_conditions is not None:
-            raise ValueError(
-                "You passed both, prepared states to resume a simulation and initial "
-                "conditions, which is not possible. Either resume a simulation or "
-                "start a new one."
-            )
```

Comment:
Why do we not need this ValueError anymore?

Reply:
The `initial_conditions` are never `None` since they will be replaced by a default dictionary if `None` is passed by the user.